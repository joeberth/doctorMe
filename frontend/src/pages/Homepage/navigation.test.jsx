import React from 'react';
import Navigation from './navigation';
import renderer from 'react-test-renderer';


test("Snapshot navigation without crash", () => {
    const component= renderer.create(
        <Navigation/>
    )
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})