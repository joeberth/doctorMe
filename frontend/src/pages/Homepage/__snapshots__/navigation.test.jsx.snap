// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot navigation 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top"
  id="menu"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <button
        className="navbar-toggle collapsed"
        data-target="#bs-example-navbar-collapse-1"
        data-toggle="collapse"
        type="button"
      >
         
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
         
        <span
          className="icon-bar"
        />
         
        <span
          className="icon-bar"
        />
         
        <span
          className="icon-bar"
        />
         
      </button>
      <a
        className="navbar-brand page-scroll"
        href="#page-top"
      >
        Doctor ME
      </a>
       
    </div>
    <div
      className="collapse navbar-collapse"
      id="bs-example-navbar-collapse-1"
    >
      <ul
        className="nav navbar-nav navbar-right"
      >
        <li>
          <a
            className="page-scroll"
            href="#features"
          >
            Funcionalidades
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#about"
          >
            Sobre
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#services"
          >
            Serviços
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#portfolio"
          >
            Galeria
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#contact"
          >
            Contato
          </a>
        </li>
        <li>
          <a
            href="signin"
          >
            Login
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;

exports[`Snapshot navigation without crash 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top"
  id="menu"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <button
        className="navbar-toggle collapsed"
        data-target="#bs-example-navbar-collapse-1"
        data-toggle="collapse"
        type="button"
      >
         
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
         
        <span
          className="icon-bar"
        />
         
        <span
          className="icon-bar"
        />
         
        <span
          className="icon-bar"
        />
         
      </button>
      <a
        className="navbar-brand page-scroll"
        href="#page-top"
      >
        Doctor ME
      </a>
       
    </div>
    <div
      className="collapse navbar-collapse"
      id="bs-example-navbar-collapse-1"
    >
      <ul
        className="nav navbar-nav navbar-right"
      >
        <li>
          <a
            className="page-scroll"
            href="#features"
          >
            Funcionalidades
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#about"
          >
            Sobre
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#services"
          >
            Serviços
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#portfolio"
          >
            Galeria
          </a>
        </li>
        <li>
          <a
            className="page-scroll"
            href="#contact"
          >
            Contato
          </a>
        </li>
        <li>
          <a
            href="signin"
          >
            Login
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
